src/components/Sidebar/Playlist.jsx

```diff
@@ -0,0 +1,24 @@
+import { PropTypes } from 'prop-types';
+
+import { CoverImage } from '../CoverImage';
+import { AlbumName } from '../AlbumName';
+import { TracksInfo } from '../TracksInfo';
+
+export const Playlist = ({ playlist }) => {
+  return (
+    <div className="playlist-container">
+      <CoverImage img={playlist.images[0].url} name={playlist.name} />
```

Comment:
in this case, we can't used `find` like we do to display the cover image for `Albums` because our fake data only has 1 item in the array and nothing we can filter by.